### PR TITLE
Dummy PT to test CI

### PR DESCRIPTION
**Pull request by ifireball**
Do NOT merge!

### CODE REVIEW

**Comment by ifireball** [comment]
ci test please

**Comment by ifireball** [comment]
ci :(\+1|thumbsup):

**Comment by ifireball** [comment]
Ok, these are failing on missing stc-ci scripts, @machacekondra is there a PR that adds those?

**Comment by ifireball** [comment]
ci test please

**Comment by machacekondra** [comment]
> Ok, these are failing on missing stc-ci scripts, @machacekondra is there a PR that adds those?

I even don't know what is that and what is it for, so no. Can you please point me to some wiki?

**Comment by machacekondra** [comment]
Nice avatar btw... ;)

**Comment by ifireball** [comment]
@machacekondra hmm... the avatar thing is gonna be confusing... ;)

About STD-CI: http://ovirt-infra-docs.readthedocs.io/en/latest/CI/Build_and_test_standards/

Do note that the jobs gonna behave strangely for now, because what we have is a mix of what Lago uses with its full oVirt-independent release pipeline and an attept to standardise it for oVirt projects.

Basically you need to comment on the PR to make anything run (Including check-merged! and it will test the PR code instead of the merged code!).

Bottom line - I probably need to write new GitHub support code to make things be really useful.

**Comment by machacekondra** [comment]
So there will be STD-CI: https://github.com/oVirt/ovirt-ansible/pull/22

**Comment by machacekondra** [comment]
ci test please
